src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -1582,6 +1596,11 @@ private static void AddMatchingActiveNodes(ref List<KeyValuePair<SyntaxNode, Syn
             ImmutableArray<SyntaxNode> oldStateMachineSuspensionPoints,
             ImmutableArray<SyntaxNode> newStateMachineSuspensionPoints)
         {
+            if (SupportsStateMachineUpdates)
```

Comment:
Nit: This feels like it should be an assert here, and the caller should check it first. Or rename to "TryAdd" maybe?

Though I don't feel too strongly about it.

Reply:
The whole method will be removed by the next PR, which implements VB support.